tests/test_lab_download.py

```diff
@@ -62,19 +62,3 @@ def test_oci_download(self, mock_oci_download, cli_runner: CliRunner):
         )
         assert result.exit_code == 0
         mock_oci_download.assert_called_once()
-
-    def test_oci_download_repository_error(self, cli_runner: CliRunner):
-        result = cli_runner.invoke(
-            lab.ilab,
-            [
-                "--config=DEFAULT",
-                "model",
-                "download",
-                "--repository=docker://quay.io/ai-lab/models/granite-7b-lab:latest",
-            ],
-        )
-        assert result.exit_code == 1
-        assert (
-            "Invalid repository supplied: Please specify tag/version 'latest' via --release"
-            in result.output
-        )
```

Comment:
Should this test not remain?

Reply:
so if anything i could keep it and make it not expect an error 
but i was wondering how much value that really adds

im ok with updating it and keeping it as well

Reply:
I would prefer to keep and update just to be something that's valid

Maybe `test_oci_download_repository_with_tag` or something like that